### PR TITLE
Update SMIRNOFF example

### PR DESCRIPTION
**Pull request by mattwthompson**
While thinking about a workflow that combines that OpenFF and MoSDeF stacks, I realized that @ahy3nz already created one. It's a little out of date - a lot can change in two years - but most of my changes were updates, not new additions.

### CODE REVIEW

File: smirnoff_omm/smirnoff.ipynb [hunk 1]
```diff
@@ -4,27 +4,35 @@
    "cell_type": "markdown",
```
Comment (daico007) [inline smirnoff_omm/smirnoff.ipynb:4]: <pre class="ql-syntax" spellcheck="false">Line #13.    from simtk import openmm, unit
</pre><p>Should we update the import syntax for openmm here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mosdef-hub/mosdef-workflows/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (justinGilmer) [inline smirnoff_omm/smirnoff.ipynb:4]: <p>This should be updated to 55.7kcal/mol</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mosdef-hub/mosdef-workflows/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (justinGilmer) [inline smirnoff_omm/smirnoff.ipynb:4]: <p>I need to inspect this further, but why are the values for oplsaa double what they were before?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mosdef-hub/mosdef-workflows/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (mattwthompson) [reply smirnoff_omm/smirnoff.ipynb:4]: Yes - but we need to make a new toolkit release for that to happen. I don't have an ETA on that, but we should probably not merge this until then.

Comment (mattwthompson) [reply smirnoff_omm/smirnoff.ipynb:4]: No clue ... that's concerning.

**Comment by mattwthompson** [comment]
Let me know if you'd like this revamped or re-written at some point ... for now I'm going to have to close this as it's grown out of date.

**Pull request closed without merging**